File: python_ToSXs/python/06-thaphinh/14.py
height = 5

shape = ""
for i in range(1, height + 1):
    space = " " * (height - i)

    left = [str(j) for j in range(i, 1, -1)]
    left = "".join(left)
    right = [str(j) for j in range(2, i + 1)]
    right = "".join(right)

    record = space + left + "1" + right + space
    shape += record + "\n"

print(shape)

File: cpp_e9zMZ/cpp/06-thaphinh/14.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    int number = 1;
    int height = 5;
    string shape = "";

    for (int i = number; i <= height; i++) {
        string space = myRepeat(" ", height - i);
        string left = createListNumberReverse(i, 2);
        string right = createListNumber(2, i);
        string record = space + left + "1" + right + space;
        shape += record + "\n";
    }
    
    cout << shape << "\n";
}